manifest.json

```diff
@@ -14,7 +14,26 @@
     "postreleasy": "vtex publish --verbose"
   },
   "dependencies": {
-    "vtex.render-runtime": "8.x"
+    "vtex.render-runtime": "8.x",
+    "vtex.apps-graphql": "2.x"
+  },
+  "settingsSchema": {
+    "title": "Structured data",
+    "type": "object",
+    "properties": {
+      "decimals": {
+        "title": "Number of decimals",
+        "type": "number",
+        "default": 2,
+        "description": "Set the number of decimals you want the prices to show"
+      },
+      "pricesWithTax": {
+        "title": "Prices with tax",
+        "type": "boolean",
+        "default": false,
+        "description": "Use the Price with Tax instead of spotPrice"
+      }
```

Comment:
You should use message ids here to make titles and descriptions i18n supported

Reply:
Sure, but don't know how to do that in manifest. I know how to in a tsx file using react-intl, but can't find anything on using the message builder and the message ids in a settingsSchema file (https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-4-configuringyourappsettings) - can you show an example of how to set the descriptions and titles in this case to allow multiple languages?

Reply:
Yep, check how we did this in our `vtex.store` app -> https://github.com/vtex-apps/store/blob/master/manifest.json#L39.

You just reference the message IDs in the same way you would inside a component

Reply:
@victorhmp - I noticed that when I add the message ids in manifest they don't display properly in admin, I tried in other apps of mine with admin builder and the same happens.  What's different between this app and store app that seems to be using the messages from admin-pages

![image](https://user-images.githubusercontent.com/71461884/127622485-a132906a-4a13-4471-9c79-12a3814570ed.png)